### ISSUE TITLE
MultiVariateKernelRegression Classification Decide/Compute Winner

### ISSUE THREAD

**Issue by blaisexen** [accord-net/framework#1243]
- [x] question

//MultivariateKernelRegression

//inputs was already transform by BoW
//youtputs is an index from 0 to Up
mvkr = kpca.Learn(inputs, youtputs);

?
How would we call to Decide which one is the Winner

mvkr.DECIDE is nothing/gone?

**Comment by fdncred** [comment]
Looking at the [code ](https://github.com/accord-net/framework/blob/837fb3363f17a5a7fe89d9046af2ae9ef7502c66/Sources/Accord.Statistics/Models/Regression/Nonlinear/MultivariateKernelRegression.cs)it seems that Transform is the method you want to call.

**Comment by blaisexen** [comment]
@fdncred 

yes thank you, but I already know transform is the method for the result.

but this is different, because I'm looking for a classification,

I have used rbm/svm then they get the winner by Decide or Compute,

//RBM
answer = xnetwork.Compute(xinput);
//SVM
int sactual = ksvm.Decide(xinput);
//
and others as well using Computer or Decide,

?
now I want to test kpca with Mvkr, but I can't determined which way to get the result.
any other idea than Transform?

**Comment by fdncred** [comment]
Unfortunately, I don't see any examples that would help. I see kpca has Learn() and Compute() but I'm not sure how to use it. Sorry.

**Comment by blaisexen** [comment]
yes, I understand, I have used like this:
MvKr = kpca.Learn(inputs, youtputs);
but I don't see MvKr property name of Compute/Decide,

thanks for the reply
